modules/plot/main/coffee/Sparkline.coffee

```diff
@@ -37,7 +37,7 @@ class Sparkline
     hx.components.clear(selector)
     hx.component.register(selector, this)
 
-    graph = new hx.Graph(selector)
+    graph = new hx.Graph(selector, options)
```

Comment:
I'm not sure what the additional options that are specific for sparkline will do if passed into the graph directly, I'd suggest adding an option for `graphOptions` and passing that through instead? That way it's a bit more explicit (we use this method for components added to Form Builder)